src/domUpdates.js

```diff
@@ -100,8 +101,18 @@ class DomUpdates {
     document.querySelector('.modal-opacity').remove();
   }
 
-  closeGroceryModal(ingredients) {
-    // console.log(ingredients);
+  closeGroceryModal(ingredients, user) {
+    let controller = new DatabaseController();
+
+    ingredients[0].forEach(ingredient => {
```

Comment:
What exactly is ingredients[0] doing here? Why is it hardcoded?(looks to be)

Reply:
the ingredients array getting instantiated in grocery list has the quantity and cost as separate items in the array. the [0] here is just getting the list of ingredients themselves.